analyzers/IsolationAnalyzer.py

```diff
@@ -6,22 +6,22 @@
 pdgids = [211, 22, 130, 11, 13]
 
 class IsolationAnalyzer(Analyzer):
-    '''Compute lepton isolation.  
+    '''Compute candidate isolation.  
```

Comment:
I understand this change, we also want to isolate photons, for example.
please avoid the use of "candidate" which is too CMS centric and not explicit enough. 
Instead, use particle (as in heppy.particles)